test/jpypetest/test_conversion.py

```diff
@@ -144,3 +144,9 @@ def testCharBytes(self):
         with self.assertRaises(TypeError):
             ja[0:1] = [object()]
         ja[0:1] = ["\u265E"]
+
+    def testDerived(self):
+        # Test Java Object any conversion
+        self.assertEqual(jpype.JClass("java.lang.Object")._canConvertToJava(jpype.JClass("java.lang.String")("a")), "derived")
```

Comment:
Is there actually another way to test this. I would like to avoid testing private implementation (even though this interface hasn't changed in a decade).